bazel/repository_locations.bzl

```diff
@@ -1165,12 +1165,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         # NOTE: Update together with com_googlesource_chromium_base_trace_event_common.
         # Patch contains workaround for https://github.com/bazelbuild/rules_python/issues/1221
         version = "10.7.193.13",
-        # Static snapshot created using https://storage.googleapis.com/envoyproxy-wee8/wee8-fetch-deps.sh.
-        sha256 = "2170df76ce5d7ecd7fb8d131370d210152f200273cba126f06d8b88fb53c9fbc",
-        urls = ["https://storage.googleapis.com/envoyproxy-wee8/v8-{version}.tar.gz"],
+        # Follow this guide to pick next stable release: https://v8.dev/docs/version-numbers#which-v8-version-should-i-use%3F
+        strip_prefix = "v8-{version}",
+        sha256 = "6fb91b839e9c36ca4c151268f772e7a6a888a75bcb947f37be9758e49f485db7",
+        urls = ["https://github.com/v8/v8/archive/refs/tags/{version}.tar.gz"],
         use_category = ["dataplane_ext"],
         extensions = ["envoy.wasm.runtime.v8"],
-        release_date = "2022-10-12",
+        release_date = "2022-09-28",
```

Comment:
@yanavlasov The SHA and the date changes are expected?

Reply:
Yes. The archives were created differently and on different dates so the checksum and date differences are expected.